js/islandora_image_annotation_dialog.js

```diff
@@ -316,9 +316,21 @@
         svgEncoding.setAttribute('content', 'utf-8');
         svgAbout.appendChild(svgEncoding);
       });
-      // XXX: Need to handle the output this way because of IE browser
-      // incompatabilities.
-      return $(xmldoc.documentElement).html();
+      
+      // Test browser features and call correct to string method
+      var serializeXmlComp = function(xmldoc) {
+        if (typeof window.XMLSerializer != "undefined") {
+          return (new window.XMLSerializer()).serializeToString( xmldoc );
+        } else if (typeof xmldoc.xml != "undefined") {
+          return xmldoc.xml;
+        }
+        return "";
+      }
+ 
+      // convert XML DOM document to string
+      var xmlAsString = serializeXmlComp( xmldoc );
+
+      return xmlAsString;
```

Comment:
Doesn't seem to cause any problems with Chrome, et all:

https://developer.mozilla.org/en/docs/XMLSerializer

This would be the chunk responsible for IE 8 compatibility?
```PHP
if (typeof xmldoc.xml != "undefined") {
  return xmldoc.xml;
}
```

Not sure why:

```PHP
$(xmldoc.documentElement).html();
```

Wouldn't work with IE? And I can't for the life of me find any reliable documentation on:

```javascript
ActiveXObject("Microsoft.XMLDOM");
```